src/user/UserCredentials.tsx

```diff
@@ -453,6 +601,50 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
           </Text>
         </Modal>
       )}
+      {openCredentialReset && (
+        <Modal
+          variant={ModalVariant.small}
+          width={900}
+          style={{ height: "20rem" }}
```

Comment:
I'm guessing you added this to make room for the dropdown, you can use it like this:
```suggestion
          variant={ModalVariant.medium}
```
And add:
```
menuAppendTo="parent"
```
to the `CredentialsResetActionMultiSelect` => `Select` then it pops out of the dialog when there is not enough place:
https://www.patternfly.org/v4/components/modal#with-dropdown